### PR TITLE
import navigation bar setup

### PR DESCRIPTION
**Pull request by alexandrashaw10**

### CODE REVIEW

File: app/src/main/java/com/example/una/MapsViewFragment.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.example.una;
+
+import android.support.v4.app.Fragment;
+
+public class FragmentMapsView extends Fragment {
```
Comment (chiped) [inline app/src/main/java/com/example/una/MapsViewFragment.java]: Does this compile? Pretty sure the name of the public class should match the name of the file

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,7 @@
 <resources>
     <string name="app_name">Una</string>
+    <string name="text_home">Home</string>
```
Comment (chiped) [inline app/src/main/res/values/strings.xml]: Nice!

File: app/src/main/java/com/example/una/MainActivity.java [hunk 3]
```diff
@@ -1,13 +1,54 @@
...
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        final FragmentManager fragmentManager = getSupportFragmentManager();
+
+        // TODO change 1st fragment to Denize's HomePageFragment
```
Comment (chiped) [inline app/src/main/java/com/example/una/MainActivity.java]: create an issue to better track this todo

**Comment by chiped** [comment]
Since you modified UI, can you add a screenshot of what it looks like? It will be easier to visualize your change

File: app/build.gradle [hunk 4]
```diff
@@ -26,4 +26,7 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+
+    implementation 'com.android.support:design:28.0.0'
+    implementation "com.android.support:recyclerview-v7:28.0.0"
```
Comment (chiped) [inline app/build.gradle]: let's stay consistent with use of quotes in this file and use `'`.

File: app/src/main/java/com/example/una/MainActivity.java [hunk 5]
```diff
@@ -1,13 +1,54 @@
...
+                        fragment = quickDonateFragment;
+                        break;
+                    case R.id.action_maps_view:
+                        fragment = mapsViewFragment;
+                        break;
+                    case R.id.action_impact:
+                        fragment = impactFragment;
+                        break;
+                    default:
+                            fragment = homePageFragment;
```
Comment (chiped) [inline app/src/main/java/com/example/una/MainActivity.java]: nitpicking at this point: indent this line consistently with the rest of the file so it is easier to read. you could select all file and hit `control`+`option`+`i` to auto indent

**Comment by alexandrashaw10** [comment]
BottomNavBar1

Thanks for the feedback!

**Pull request merged**